content/guides/getting-started/styling-components.md

```diff
@@ -51,7 +52,7 @@ What you would see is classes being applied in the inspector, but no styles. e.g
 
 PopperJS (Floating UI) is probably a better example because the UI is malformed -- not just missing. e.g. the :after CSS being semi-visible
 -->
-
+<!-- see GitHub comment -->
 Components can have three parts: markup, styles, and script logic. All three of
 these work together in order to deliver a working component.
```

Comment:
The practical guidelines seem to be mixed in with educational content around component testing or the benefits of Cypress. 

For someone who is already familiar with Cypress and the benefits of component testing, it can feel like they have to read a lot to get to the info they're looking for. 

For example, I would have expected the section on [3rd Party CSS Libraries (Tailwind, Bootstrap, PopperJS)](https://deploy-preview-4186--cypress-docs.netlify.app/guides/getting-started/styling-components#3rd-Party-CSS-Libraries-Tailwind-Bootstrap-PopperJS) to tell me how to load those libraries, but it's purely educational